src/runtime/virtcontainers/hypervisor.go

```diff
@@ -386,6 +386,24 @@ type HypervisorConfig struct {
 	// TxRateLimiterMaxRate is used to control network I/O outbound bandwidth on VM level.
 	TxRateLimiterMaxRate uint64
 
+	// NetRateLimiterBwRate is used to control network I/O bandwidth on VM level.
+	// The same value, defined in bits per second, is used for inbound and outbound bandwidth.
+	NetRateLimiterBwMaxRate uint64
+
+	// NetRateLimiterBwOneTimeBurst is used to control network I/O bandwidth on VM level.
+	// This increases the initial max rate and this initial extra credit does *NOT* replenish
+	// and can be used for an *initial* burst of data.
+	NetRateLimiterBwOneTimeBurst uint64
+
+	// NetRateLimiterOpsRate is used to control network I/O bandwidth on VM level.
```

Comment:
s/bandwidth/operations per second/ ?

If so then the same applies to `NetRateLimiterOpsOneTimeBurst` below and their equivalents for disk I/O.

Given that "bandwidth" and "operations per second" are clearly distinct concepts in this PR I think it's worth the trouble to avoid any risk of mixing them up.

Reply:
Very nice catch!